### PR TITLE
Fix TOTP login validation

### PR DESCRIPTION
**Pull request by therealssj**
Started removing the context. 
Now you can setup TOTP and use it for validation when logging in.

### CODE REVIEW

File: src/Form/BasicSetup.php [hunk 1]
```diff
@@ -134,15 +133,15 @@ public function buildForm(array $form, FormStateInterface $form_state, User $use
           $context['setup_context'] = ['plugin_definition' => $plugin_definition];
           $form['#title'] = t('TFA setup - Trusted browsers');
           $setup_plugin = new TfaTrustedBrowserSetup($context);
-          $tfa_setup = new TfaSetup($setup_plugin, $context);
+          $tfa_setup = new TfaSetup($setup_plugin);
```
Comment (nerdstein) [inline src/Form/BasicSetup.php:136]: If we removed $context, why are we building it above? see line 133 as an example

Comment (nerdstein) [inline src/Form/BasicSetup.php:136]: Ah, disregard, I see line 135

File: src/Form/BasicSetup.php [hunk 2]
```diff
@@ -134,15 +133,15 @@ public function buildForm(array $form, FormStateInterface $form_state, User $use
           $context['setup_context'] = ['plugin_definition' => $plugin_definition];
           $form['#title'] = t('TFA setup - Trusted browsers');
           $setup_plugin = new TfaTrustedBrowserSetup($context);
```
Comment (nerdstein) [inline src/Form/BasicSetup.php:135]: Over time, I'm assuming this "setup" plugin will be built into the configuration on top of the plugin systems. Is this correct? I see "trusted browser", but how will this evolve later with the plugin systems?

File: src/Form/BasicSetup.php (hunk 1 above)
Comment (therealssj) [inline src/Form/BasicSetup.php:136]: Those are for other plugins I havent tackled those yet.
If I removed it the module would break.

**Comment by nerdstein** [comment]
Reviewed and merging

**Pull request merged**